shell/platform/fuchsia/runtime/dart/utils/BUILD.gn

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 The Fuchsia Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+assert(is_fuchsia)
+
+import("//build/fuchsia/sdk.gni")
+
+config("utils_config") {
+  include_dirs = [ "../../.." ]
+}
+
+source_set("utils") {
+  sources = [
+    "files.cc",
+    "files.h",
+    "handle_exception.cc",
+    "handle_exception.h",
+    "inlines.h",
+    "logging.h",
+    "tempfs.cc",
+    "tempfs.h",
+    "vmo.cc",
+    "vmo.h",
+    "vmservice_object.cc",
+    "vmservice_object.h",
+  ]
+
+  deps = [
+    "$fuchsia_sdk_root/fidl:fuchsia.crash",
+    "$fuchsia_sdk_root/fidl:fuchsia.mem",
+    "$fuchsia_sdk_root/pkg:async-loop",
+    "$fuchsia_sdk_root/pkg:async-loop-cpp",
+    "$fuchsia_sdk_root/pkg:fdio",
+    "$fuchsia_sdk_root/pkg:memfs",
+    "$fuchsia_sdk_root/pkg:syslog",
+    "$fuchsia_sdk_root/pkg:zx",
+    "$fuchsia_sdk_root/pkg/lib/sys/cpp",
+    "$fuchsia_sdk_root/pkg/lib/vfs/cpp",
+    "//third_party/tonic",
+  ]
+
+  public_configs = [ ":utils_config" ]
+}
```

Comment:
This drops the package rule for run_vmservice_object_tests. We need to ensure that that tests continues to run after the migration.

Reply:
That rule refers to the non-existent (yet) template definitions in `//build/package.gni`. I was going to build up those rules in subsequent packages. This rule just works builds what is necessary for existing targets.

May I add those rules in subsequent patches after I port the template definitions?

Reply:
sgtm